File: Marks/Marks.js
// Check Grade Accourding to your marks:

let num = parseInt(prompt("Enter marks to check: "));

if (isNaN(num)) {
    console.log("You entered wrong input. Please enter correct input 😵😕");
}
else if (num >= 91 && num <= 100) {
    console.log(`num = ${num}. You got A+ Grade 🤩✌️`);
}
else if (num >= 81 && num <= 90) {
    console.log(`num = ${num}. You got A Grade 🤗🎉`);
}
else if (num >= 71 && num <= 80) {
    console.log(`num = ${num}. You got B+ Grade 😍👌`);
}
else if (num >= 61 && num <= 70) {
    console.log(`num = ${num}. You got B Grade ☺👋`);
}
else if (num >= 33 && num <= 60) {
    console.log(`num = ${num}. You got C Grade 🙂🙆`);
}
else if (num < 33 && num >= 0 ) {
    console.log(`num = ${num}. You Failed! 😭🥹`);
}
else {
    console.log("Invalid marks. Please enter marks between 0 and 100.");
}

File: Marks/Marks.py
# Check Grade Accourding to your marks:

try:
    num = int(input("Enter marks: "))

    if num > 100 or num < 0:
        print("Invalid marks. Please enter marks between 0 and 100.")
    elif 91 <= num <= 100:
        print(f"num = {num}. You got A+ Grade🤩✌️")
    elif 81 <= num <= 90:
        print(f"num = {num}. You got A Grade🤗🎉")
    elif 71 <= num <= 80:
        print(f"num = {num}. You got B+ Grade😍👌")
    elif 61 <= num <= 70:
        print(f"num = {num}. You got B Grade☺👋")
    elif 33 <= num <= 60:
        print(f"num = {num}. You got C Grade🙂🙆")
    else:
        print(f"num = {num}. You Failed😭🥹")
except ValueError:
    print("You entered Wrong Input😵😕")
